src/Invoker.ts

```diff
@@ -50,7 +50,11 @@ export class Invoker {
             };
 
             try {
-                googleFunction(jsonRequest, response);
+                const promisifiedResponse = googleFunction(jsonRequest, response);
+                if (promisifiedResponse && promisifiedResponse.then) {
```

Comment:
Not sure if that is the best why to check if is a promise or not https://stackoverflow.com/questions/27746304/how-do-i-tell-if-an-object-is-a-promise

Reply:
@ecruzado agreed, changed it to use Promise.resolve as specified in the thread